assets/queries-handler.js

```diff
@@ -53,19 +56,60 @@ if (searchTitle) {
   searchTitle.innerHTML = query;
 }
 
-if (sortSelect) {
+// dropdown select
+if (dropdown) {
   let sortField = urlParams.get('sort_field');
-  
+  let sortOrder = urlParams.get('sort_order');
+
   if (!sortField) {
     // Set a default value for sortField if it is null or undefined
-    sortField = sortSelect.options[0].value;
+    sortField = 'price';
+  }
+
+  if (!sortOrder) {
+    // Set a default value for sortOrder if it is null or undefined
+    sortOrder = 'asc';
   }
```

Comment:
This can be shortened to 
``` 
const sortField = urlParams.get('sort_field') || 'price';
const sortOrder = urlParams.get('sort_order') || 'asc';
 ```